.github/workflows/rust.yml

```diff
@@ -15,12 +15,19 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
```

Comment:
I'm working to tidy up the CI across this family of repos, we will need to rebase that once it is in place; trying to finish in the next 6 hours or so

Reply:
Great! I rebased and adding building no-std. https://github.com/ralexstokes/ssz-rs/pull/25/files#diff-73e17259d77e5fbef83b2bdbbe4dc40a912f807472287f7f45b77e0cbf78792dR31